src/main/webapp/style.css

```diff
@@ -26,6 +26,7 @@ ul {
 }
 
 .navigation {
+  display: flex;
   margin-left: auto;
```

Comment:
nit (and not the scope of this PR): use the shorthand property margin

Reply:
I think it can be adjusted here since changes were made to the navigation's styling.

Reply:
Replaced here; Scope of PR adjust accordingly 64a7f58